src/stk/molecular/topology_graphs/metal_complex/paddlewheel/paddlewheel.py

```diff
@@ -0,0 +1,93 @@
+"""
+Paddlewheel
+===========
+
+"""
+
+from ..metal_complex import MetalComplex
+from ..vertices import _MetalVertex, _BiDentateLigandVertex
+from ...topology_graph import Edge
+
+
+class Paddlewheel(MetalComplex):
+    """
+    Represents a metal complex topology graph.
+
+    Metal building blocks with at least four functional groups are
+    required for this topology.
+
+    Ligand building blocks with two functional groups are required for
+    this topology.
+
+    :class:`.BuildingBlock` placements:
+        metals: (0, 1)
+        ligands: (0, 1, 2, 3)
+
+    See :class:`.MetalComplex` for more details and examples.
```

Comment:
Maybe put this into a See Also section.

Reply:
I took this straight from your cage topologies.

Reply:
Proof I'm not perfect.